io.gitlab.celleron56.libretile.yaml

```diff
@@ -0,0 +1,92 @@
+app-id: io.gitlab.celleron56.libretile
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: /app/files/start.sh
+finish-args:
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=all
+  - --share=network
+
+
+modules:
+  - name: libretile
```

Comment:
since this is the main module, it should be last.

Reply:
the reason it is not last is because if it where last i would need to wait for tk and pillow to build bevore i could check if the libretile module worked (since  i know that the tkinter and pillow modules  work)

i'll try to move it  (i'll test it locally first )